lib/ConvoyLeader/src/App.cpp

```diff
@@ -132,6 +249,32 @@ void App::loop()
  * Private Methods
  *****************************************************************************/
 
+void App::positionTopicCallback(const String& payload)
+{
+    StaticJsonDocument<JSON_DOC_DEFAULT_SIZE> jsonPayload;
+    DeserializationError                      error = deserializeJson(jsonPayload, payload.c_str());
+
+    if (error != DeserializationError::Ok)
+    {
+        LOG_ERROR("JSON Deserialization Error %d.", error);
+    }
+    else
+    {
+        JsonVariant xPos = jsonPayload["X"];

```

Comment:
Best practice is to have "json" as postix, e.g. "jsonXPos" ... so its easier to see that its a JSON object.